manifests/n/Nextcloud/NextcloudDesktop/3.13.2/Nextcloud.NextcloudDesktop.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Nextcloud.NextcloudDesktop
+PackageVersion: 3.13.2
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: Reboot=ReallySuppress
+InstallerSuccessCodes:
+- 3010
+UpgradeBehavior: install
+ProductCode: '{A9593531-5BE7-4772-848B-58F4F3AA06CA}'
+ReleaseDate: 2024-07-01
+AppsAndFeaturesEntries:
+- DisplayVersion: 3.13.1.20240701
+  UpgradeCode: '{FD2FCCA9-BB8F-4485-8F70-A0621B84A7F4}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/nextcloud-releases/desktop/releases/download/v3.13.1/Nextcloud-3.13.1-x64.msi

```

Comment:
>  Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [3.13.1.20240701, 3.13.1.20240701]]

Hi @PckgrBot, 

These values seem to be from the previous version of the package. Is this accurate?